content/guides/pedestal-with-component.adoc

```diff
@@ -1,16 +1,419 @@
 = Using Pedestal With Component
-Michael Nygard
-2016-08-26
+Daniel De Aguiar
+2016-09-11
 :jbake-type: page
 :toc: macro
 :icons: font
 :section: reference
 
 ifdef::env-github,env-browser[:outfilessuffix: .adoc]
 
-== Placeholder
+toc::[]
+_by {author}_, _{revdate}_
 
-Pull requests welcomed!
+== What You Will Learn
+After reading this guide you will be able to:
 
+- Create a Component-based service using Pedestal.
+- Test your service using Pedestal's test helpers.
 
-__TODO__ Incorporate information from https://github.com/pedestal/pedestal/issues/366
+== Guide Assumptions
+This guide is for users who are familiar with Clojure, Pedestal,
+Component and Boot. If you are new to Pedestal, you may want to go back to the
+link:hello-world[Hello World guide]. If you're new to Component, you
+should definitely https://github.com/stuartsierra/component[check it
+out] first.
+
+== Getting Help if You're Stuck
+If you get stuck at any point in this guide, please submit an
+https://github.com/pedestal/docs/issues[issue] about this guide or hop
+over to the
+https://groups.google.com/forum/#!forum/pedestal-users[mailing list]
+and raise your hand there.
+
+This guide shows fragments of code as we add them. Sometimes it helps
+to see the whole thing at once, so you can always check out
+<<_the_whole_shebang,the whole shebang>> at the end of this guide.
+
+== Where We Are Going
+In this guide, we're going to step through creating a Pedestal service
+using Component. We'll start off by creating a Pedestal component and
+wire it into a Component system map. We'll then proceed to testing our
+service.
+
+== Before We Begin
+What responsibilities do we want the Pedestal component to have? For
+the purposes of this guide, it should only be responsible for the
```

Comment:
Is this really just for the purposes of this guide? If so, why is this different than a real system?